hrui/hrui.php

```diff
@@ -934,3 +944,89 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu List of available menu items
+ */
+function _createHelpMenu(&$menu) {
+  $menu[] = array(
+    'attributes' => array(
+      'label'      => 'Help',
```

Comment:
Prevously, i was adding the parent menu directly as 
```php
$menu[] = array(
 'attributes' => array(
 'label'      => 'Help', // check if required
 'name'       => 'Help',
 'url'        => null, // check if required
 'permission' => 'access CiviCRM',
 'active'     => 1 // check if required
 )
);
```
In this case the `label`, `active` and `url` attributes are required. Bt i have changed the implementation now to use `_hrui_civix_insert_navigation_menu` to create the parent menu as well. So the respective attributes  can be removed now. Refactored the code to create menu as:
```php
 _hrui_civix_insert_navigation_menu($menu, '', array(
    'name' => ts('Help'),
    'permission' => 'access CiviCRM'
  ));
```
```php

_hrui_civix_insert_navigation_menu($menu, '', array(
    'name' => ts('Developer'),
    'permission' => 'access CiviCRM'
  ));
```